modules/ppcp-settings/resources/js/Components/ReusableComponents/ConnectionInfo.js

```diff
@@ -1,54 +1,58 @@
 import { __ } from '@wordpress/i18n';
-import { useState } from '@wordpress/element';
+import { useState, useEffect } from '@wordpress/element';
+import {CommonHooks} from "../../data";
 
-const ConnectionInfo = ( { connectionStatusDataDefault } ) => {
-	const [ connectionData, setConnectionData ] = useState( {
-		...connectionStatusDataDefault,
-	} );
 
-	const toggleStatusClassName = [ 'ppcp-r-connection-status__status-toggle' ];
+const ConnectionInfo = () => {
+    const { merchant } = CommonHooks.useMerchantInfo();
+    const [connectionData, setConnectionData] = useState(getDefaultConnectionStatusData(merchant));
```

Comment:
Let's directly use the `merchant` here object instead of adding a useState with a direct copy of the object